raceforcash/src/models/Race.ts

```diff
@@ -0,0 +1,6 @@
+export interface Race {
+    id: number;
+    name: string;
+    startTime: Date;
+    payouts: number;
```

Comment:
This should be array

Reply:
Add a horse array also

Reply:
Made payouts an array, added a Payout interface within model.